src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationServicesService.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.pip.data.management.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.reactive.function.client.WebClientException;
+
+import java.util.Map;
+
+import static org.springframework.security.oauth2.client.web.reactive.function.client.ServerOAuth2AuthorizedClientExchangeFilterFunction.clientRegistrationId;
+
+@Slf4j
+@Component
+public class PublicationServicesService {
+
+    @Autowired
+    WebClient webClient;
+
+    @Value("${service-to-service.publication-services}")
+    private String url;
+
+    public String sendNoMatchArtefactsForReporting(Map<String, String> noMatchMap) {
+        try {
+            return webClient.post().uri(url + "/notify/unidentified-blob")
+                .body(BodyInserters.fromValue(noMatchMap))
+                .attributes(clientRegistrationId("publicationServicesApi"))
+                .retrieve().bodyToMono(String.class).block();
+        } catch (WebClientException ex) {
+            return String.format("Sending map of no match artefacts: %s, "
```

Comment:
this could get burried very easily as youre returning an error into a info log, i think a potential serviceToServiceException should be thrown, or if youre not wanting to break you could create a nonEssentialServiceToServiceException, but currently any failures wouldnt be detected as theyre instantly burried

Reply:
Added a serviceToService exception like we have in publication services to handle this.

Reply:
the flow should break on this failure then?